### ISSUE TITLE
Final output json file  

### ISSUE THREAD

**Issue by apourghani** [exiawsh/StreamPETR#78]
Hello again,
In the previous issue, we asked for the output of the model for the test set, now for optimizing some of our model's parameters, we need to do an exhaustive search on them. but the NuScenes test set doesn't have annotations!
We will be very thankful if you provide the output results(json file preferred) of the model for the evaluation set.
Here's my email: apourghani@gmail.com
Thank you very much,
Aliasghar Pourghani,
Student of Sharif University of Technology.

**Comment by exiawsh** [comment]
@apourghani The StreamPETR Large is trained on the training and val set of nuscenes. There is a data leakage issue here. We don't have the bandwidth to retrain it on the training set. You can use our open-source VOV weights to obtain detection results.

**Comment by apourghani** [comment]
So is there anyway to calculate metrics like NDS and mAP metrics on test set?(we don't have ground truth obviously!)
We haven't finished our model's paper to submit to NuScene website yet.
